### ISSUE TITLE
Instanced Flags

### ISSUE THREAD

**Issue by GameDungeon** [miki151/keeperrl#1802]
Make it where you can set a flag for a certain creature or furniture instance

**Comment by miki151** [comment]
Add creature flags for now

**Comment by GameDungeon** [comment]
Any chance of getting the furniture part of this. Me Kappa and Sotona are making a factory mod, that while works without the flags the mod would be greatly improved by them.

**Comment by miki151** [comment]
Specific furniture or per position? Ex. would you set different flags for floor and table, and would the flag disappear if furniture is removed and something else is built there.

**Comment by GameDungeon** [comment]
Exactly. Per position, and yes it disappears if removed. Flags can be accessed by an effect, Either by other things, or themselves.
